pkg/controller/storagecluster/reconcile.go

```diff
@@ -23,15 +26,19 @@ import (
 	statusutil "github.com/openshift/ocs-operator/pkg/controller/util"
 	cephv1 "github.com/rook/rook/pkg/apis/ceph.rook.io/v1"
 	rook "github.com/rook/rook/pkg/apis/rook.io/v1alpha2"
-	corev1 "k8s.io/api/core/v1"
 )
 
 const (
 	nodeAffinityKey   = "cluster.ocs.openshift.io/openshift-storage"
 	nodeTolerationKey = "node.ocs.openshift.io/storage"
 )
 
-var defaultMonCount int = 3
+var validTopologyLabelKeys = []string{
+	"failure-domain.beta.kubernetes.io",
+	"failure-domain.kubernetes.io",
+}
+
+var defaultMonCount = 3
```

Comment:
Why not just assign it to monCount? Unless we are allowing it to be configurable, which is not the case currently.

Reply:
I had made the mon count configurable using an env var a couple of weeks ago.  I needed a way to override the default of '3' in order to support replica 2 for the test suite. 

If you all want to remove usage of the MON_COUNT_OVERRIDE variable now, that should be fine because the e2e tests now run with replica 3. The variable does serve as an escape hatch for us though if we never need to configure the mon count for some reason.